### PR TITLE
Adding the city of Melbourne!

### PR DESCRIPTION
**Pull request by Ivoz**
Also adjusted backgrounds' positions to look a bit nicer.

It would be nice to also adjust background-size per-city, but that's
done in a mixin currently.

### CODE REVIEW

File: _includes/_footer.html [hunk 1]
```diff
@@ -11,6 +11,9 @@
             <li>
               <a href="/stockholm/">Stockholm</a>
             </li>
+            <li>
+              <a href="/melbourne/">Stockholm</a>
```
Comment (nekopanic) [inline _includes/_footer.html]: Sure it's spelled "Stockholm", but it's pronounced "Melbourne".

File: melbourne/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+<ul class="float_list float_list_4 team_list">
+
+  <li class="member">
+    <img src="/images/team/steven.jpg">
+    <h3>Steven Farlie</h3>
+    <p>steven@opentechschool.org</p>
+  </li>
+
+  <!--<li class="member">-->
+    <!--<img src="/images/team/matthew.jpg">-->
```
Comment (nekopanic) [inline melbourne/index.md:24]: Feel free to add your headshot and uncomment this,

Comment (Ivoz) [inline melbourne/index.md:24]: Yah, will do when I get maself a nice headshot

File: _includes/_footer.html (hunk 1 above)
Comment (Ivoz) [inline _includes/_footer.html]: Cheers, good eyes! 7c91502 should fix

**Comment by nekopanic** [comment]
Looks good. I added Melbourne feeds to blog.opentechschool.org as well.

**Pull request merged**